lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml

```diff
@@ -88,14 +88,10 @@ deploymentSecurity:
   # User with UID 1000 is set in RNC docker image
   runAsUser: 1000
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   memory: 512Mi
-  #   cpu: 1000m
-  # requests:
-  #   memory: 256Mi
-  #   cpu: 500m
+resources:
```

Comment:
Too high request and too low limit.

Reply:
I set it double as default values was, due to application failure. With this resource limits, the application was running successfully, and I was able to connect the device. This resource values also can be override in helm deployment command.

If you have in mind some specific values, please write it here.

Reply:
Done